daemon/logger/syslog/syslog.go

```diff
@@ -207,3 +231,17 @@ func parseTLSConfig(cfg map[string]string) (*tls.Config, error) {
 
 	return tlsconfig.Client(opts)
 }
+
+func parseLogFormat(logFormat string) (syslog.Formatter, syslog.Framer, error) {
+	switch logFormat {
+	case "":
+		return syslog.UnixFormatter, syslog.DefaultFramer, nil
```

Comment:
@calavera but moving it under default doesn't catch the "default", it catches anything that isn't defined, so `--log-format=bogus` would use the default, instead of producing an error?